src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -146,46 +146,48 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 			// For Escape - the focused element is the div.notebook-preview or textarea.inputarea of the cell, so we need to make sure that it is a descendant of the current active cell
 			//  on the current active editor.
 			const activeCellElement = this.container.nativeElement.querySelector(`.editor-group-container.active .notebook-cell.active`);
+			const findWidgetVisible = document.querySelector(`.editor-widget.find-widget.visible`);
```

Comment:
Normally names like this imply it's a boolean. I'd just add the `isUndefinedOrNull` here so that still is as expected (you could also do `!!document.querySelector('.editor-widget.find-widget.visible');` which is a bit simpler, but whatever your preference is)